Package.swift

```diff
@@ -0,0 +1,18 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "DebugKit",
+    platforms: [
+        .iOS(.v14)
```

Comment:
I set 14 as the minimum because that's when the majority of the metrics were released, and allowed us to use list collection view layouts and reduce the amount of "boilerplate" UI for the menu.